packit_service/worker/handlers/mixin.py

```diff
@@ -232,6 +232,19 @@ class GetKojiBuildDataFromKojiServiceMultipleBranches(
     def _dist_git_branch(self) -> str:
         return self.branches[self._branch_index]
 
+    @property
+    def build(self):
+        # call it every time since dist_git_branch reference can change
+        build = self.koji_helper.get_latest_build_in_tag(
+            package=self.project.repo,
+            tag=self._dist_git_branch,
+        )
+        if not build:
+            raise PackitException(
+                f"No build found for package={self.project.repo} and tag={self.dist_git_branch}"
+            )
```

Comment:
I am not familiar with this code base: does this exception imply there is a bug in our code? meaning, if there is no such build, we should have create it?

Reply:
Yes, this mixins are mostly used in the bodhi update handlers/checkers and we should create a koji build before calling them. So probably if we don't find any build we missed something or servers are not yet updated/synced?

Reply:
oki, understood

just let's be sure that when this happens, we have all the info to diagnose the problem and figure out a solution